Sources/SwifterSwift/UIKit/UIButtonExtensions.swift

```diff
@@ -174,16 +174,53 @@ public extension UIButton {
         states.forEach { setTitle(title, for: $0) }
     }
 
-    /// SwifterSwift: Center align title text and image on UIButton
+    /// SwifterSwift: Center align title text and image
     ///
-    /// - Parameter spacing: spacing between UIButton title text and UIButton Image.
+    /// - Parameter spacing: spacing between title text and image.
     func centerTextAndImage(spacing: CGFloat) {
         let insetAmount = spacing / 2
         imageEdgeInsets = UIEdgeInsets(top: 0, left: -insetAmount, bottom: 0, right: insetAmount)
         titleEdgeInsets = UIEdgeInsets(top: 0, left: insetAmount, bottom: 0, right: -insetAmount)
         contentEdgeInsets = UIEdgeInsets(top: 0, left: insetAmount, bottom: 0, right: insetAmount)
     }
 
+    /// SwifterSwift: align title text and image vertically
+    /// - Parameters:
+    ///   - imageAboveText: set true for image above text (default is true).
+    ///   - spacing: spacing between title text and Image.
+    func alignTextAndImageVertically(imageAboveText: Bool = true, spacing: CGFloat = 6.0) {
```

Comment:
Again, you're not aligning them vertically. In fact you're doing the same as the `centerTextAndImage` except that you're not overlapping them, you're setting the vertical ordering, which is probably what the original function should have done in the first place. So I think use the same name but without default parameter values (at least for `imageAboveText`), otherwise you can end up with compiler errors
```suggestion
    func centerTextAndImage(imageAboveText: Bool, spacing: CGFloat) {
```

Reply:
I named it is shorted from: align image and text along the vertical axis, and omitted axis.
But even if my explanation is OK, I know the name is not a good name because it is easy to misunderstand.

So, how about merge the two `centerTextAndImage` methods and providing options like this:

```swift
enum ImagePosition {
    case aboveText
    case belowText
    case onTheLeftOfText
    case onTheRightOfText
}

func centerTextAndImage(image position: ImagePosition = .onTheLeftOfText, spacing: CGFloat)
```

Reply:
@qchenqizhi The problem with this approach is that specifically for the extensions on this project, we avoid creating new types, just as an overall rule to keep the extensions simpler, easy to just copy-paste ...

Reply:
OK, in my case, the most commonly used is image above text and image on the left of text, so as guykogus suggested: use a bool parameter to identify the two types?

```
func centerTextAndImage(imageAboveText: Bool = false, spacing: CGFloat)
```